src/main/java/de/medizininformatik_initiative/process/report/spring/config/ReportConfig.java

```diff
@@ -94,7 +95,14 @@ public DownloadSearchBundle downloadSearchBundle()
 	@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
 	public CheckSearchBundle checkSearchBundle()
 	{
-		return new CheckSearchBundle(api);
+		return new CheckSearchBundle(api, searchQueryCheckService());
+	}
+
+	@Bean
+	@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
```

Comment:
```suggestion
```

Is prototype scope needed? Service seems to have no persistent state. A singleton instance should be fine.

Reply:
changed with b72664366a0b71bf85d886b303618e96ee5760e2